src/common/SConscript

```diff
@@ -1,6 +1,30 @@
 """Build common libraries"""
 import daos_build
 
+COMMON_SRC = ['debug.c', 'mem.c', 'fail_loc.c', 'lru.c',
+         'misc.c', 'pool_map.c', 'sort.c', 'btree.c', 'prop.c',
+         'btree_class.c', 'tse.c', 'rsvc.c', 'checksum.c',
+         'drpc.c', 'drpc.pb-c.c', 'proc.c',
+         'acl_api.c', 'acl_util.c', 'acl_principal.c',
+         'profile.c']
+
+def build_common(env, client):
+    denv = env.Clone()
+
+    if client:
+        denv.Append(CPPDEFINES=['-DDAOS_CLIENT_BUILD'])
+        denv.Append(OBJPREFIX="v_")
```

Comment:
yay, it won't give the scons warning :-)

Reply:
Indeed.. That was pretty verbose..